### PR TITLE
feat: support eoscr-theme

### LINKED ISSUE #31: Support EOS CR theme

### PR DESCRIPTION
**Pull request by JustinCast**
# Support eoscr-theme
This PR provides support @eoscostarica/eoscr-theme npm package
 ## How should this be tested
1. Clone this branch
1. Type `yarn install` command in the terminal
1. Type `yarn start` command in the terminal
1. Go to `http://localhost:3000` and see the new changes

### Linked Issues
- closes #31

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]

File: src/theme/palette.js [hunk 1]
```diff
@@ -1,52 +1,61 @@
 import { colors } from '@material-ui/core'
-
+import { EOSCR_THEME } from '@eoscostarica/eoscr-theme'
```
Comment (tetogomez) [inline src/theme/palette.js:1]: Move this import to the top on the file, remember the order of imports and also add a blank space before variables

File: src/theme/typography.js [hunk 2]
```diff
@@ -1,103 +1,111 @@
 import palette from './palette'
 
+import { EOSCR_THEME } from '@eoscostarica/eoscr-theme'
```
Comment (tetogomez) [inline src/theme/typography.js:1]: same here!

**Review by tetogomez — COMMENTED** [review]
Check the comments

**Review by tetogomez — APPROVED** [review]
LGTM

**Pull request merged**